Tic Tac Toe.py

```diff
@@ -0,0 +1,175 @@
+        return True
+    if dic['4'] == dic['5'] == dic['6'] == sypher:
+        return True
+    if dic['7'] == dic['8'] == dic['9'] == sypher:
+        return True
+
+    if dic['1'] == dic['4'] == dic['7'] == sypher:
+        return True
+    if dic['2'] == dic['5'] == dic['8'] == sypher:
+        return True
+    if dic['3'] == dic['6'] == dic['9'] == sypher:
+        return True
+
+    if dic['1'] == dic['5'] == dic['9'] == sypher:
+        return True
+    if dic['3'] == dic['5'] == dic['7'] == sypher:
+        return True
+
+    return False
+
+def exit_requests(player1, player2):
+    for i in dic:
+        dic[i] = ' '
+    if player1 == 1:
+        print('Player2 `s turn')
+        return (0, 1)
+    elif player2 == 1:
+        print('Player1 `s turn')
+        return (1, 0)
+
+def main():
+    while True:
+        next_game = True
+        while next_game:
+            next_game = False
+            print_template()
+            player1, player2 = first_player()
+            mark_player1, mark_player2 = choose_sign(player1, player2)
+            score = {'Player1': 0, 'Player2': 0}
+            count = 0
+            while count < 3:
+                count += 1
+                while True:
+                    if player1 == 1:
+                        try:
+                            inpt = input('Player1 please insert position: ')
+                            if dic[inpt] == ' ':
+                                dic[inpt] = mark_player1
+                                print_progress(dic)
+                                if win(dic, mark_player1):
+                                    print('Player1 is winner')
+                                    score['Player1'] += 1
+                                    break
+                            else:
+                                print('Possition is occupied ')
+                                continue
+                        except :
+                            print('You have to insert integer 1-9 !')
+                            continue
+                        player1, player2 = 0, 1
+
+                    elif player2 == 1:
+                        try:
+                            inpt = input('Player2 please insert position: ')
+                            if dic[inpt] == ' ':
+                                dic[inpt] = mark_player2
+                                print_progress(dic)
+                                if win(dic, mark_player2):
+                                    print('Player2 is winner')
+                                    score['Player2'] += 1
+                                    break
+                            else:
+                                print('Possition is occupied ')
+                                continue
+                        except :
+                            print('You have to insert integer 1-9 !')
+                            continue
+                        player1, player2 = 1, 0
```

Comment:
Again, you should not duplicate code. [Don't Repeat Yourself](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself)